multiraft/transport.go

```diff
@@ -118,12 +118,10 @@ func (lt *localRPCTransport) accept(server *rpc.Server, listener net.Listener) {
 	for {
 		conn, err := listener.Accept()
 		if err != nil {
-			if strings.HasSuffix(err.Error(), "use of closed network connection") {
-				return
+			if !strings.HasSuffix(err.Error(), "use of closed network connection") {
+				// TODO(bdarnell): are any transient errors possible here?
+				log.Fatalf("localRPCTransport.accept: %s", err)
```

Comment:
I don't think you want to Fatal here. Vaguely remember going over this with Ben.